src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -230,9 +230,15 @@ public ContingencyContext getContingencyContext() {
         public EquationTerm<V, E> getFunctionEquationTerm() {
             switch (functionType) {
                 case BRANCH_ACTIVE_POWER:
```

Comment:
Do we have to keep case `BRANCH_ACTIVE_POWER`, I though the compatibility was done at a upper level?